FirstApp/ViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ViewController.swift
+//  FirstApp
+//
+//  Created by Reddy, P on 01/05/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController{
+
+    @IBOutlet weak var textFeild: UITextField!
+    @IBOutlet weak var output: UILabel!
+    @IBOutlet weak var slider: UISlider!
+    @IBOutlet weak var button: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        textFeild.delegate=self
+        // Do any additional setup after loading the view.
+    }
+
+    @IBAction func submit(_ sender: UIButton) {
+        let utext = textFeild.text
+        output.text = utext
+    }
+    
+    @IBAction func slide(_ sender: Any) {
+        let slider = sender as!UISlider
+        let val = (Int(slider.value))
+        output.text = "\(val)"
```

Comment:
We don't have stepper instance and we are not notifying the stepper about the slider change. Create a local variable in the Viewcontroller to store the state/value.